mappings/net/minecraft/entity/passive/AnimalEntity.mapping

```diff
@@ -3,6 +3,12 @@ CLASS net/minecraft/class_1429 net/minecraft/entity/passive/AnimalEntity
 	FIELD field_6745 loveTicks I
 	METHOD <init> (Lnet/minecraft/class_1299;Lnet/minecraft/class_1937;)V
 		ARG 1 type
+	METHOD method_20663 isValidNaturalSpawn (Lnet/minecraft/class_1299;Lnet/minecraft/class_1936;Lnet/minecraft/class_3730;Lnet/minecraft/class_2338;Ljava/util/Random;)Z
```

Comment:
Same question as above?

Reply:
I took the name from #854; if I named this as `canSpawn`, Enigma wouldn't allow me to map the subclass methods as `canSpawn`.